docs/reference/requirement-sets/dialog-api-requirement-sets.md

```diff
@@ -12,12 +12,24 @@ Requirement sets are named groups of API members. Office Add-ins use requirement
 
 Office Add-ins run across multiple versions of Office. The following table lists the Dialog API requirement sets, the Office host applications that support that requirement set, and the build or version numbers for the Office application.
 
-|  Requirement set  | Office 2013 on Windows\*<br>(one-time purchase) | Office 2016 or later on Windows\*<br>(one-time purchase)   | Office on Windows<br>(connected to a Microsoft 365 subscription) |  Office on iPad<br>(connected to a Microsoft 365 subscription)  |  Office on Mac<br>(connected to a Microsoft 365 subscription)  | Office on the web  |  Office Online Server  |
+|  Requirement set  | Office 2013 on Windows\*<br>(one-time purchase) | Office 2016 or later on Windows\*<br>(one-time purchase)   | Office on Windows<br>(subscription) |  Office on iPad<br>(subscription)  |  Office on Mac<br>(subscription)  | Office on the web  |  Office Online Server  |
 |:-----|-----|:-----|:-----|:-----|:-----|:-----|:-----|
-| DialogApi 1.1  | Build 15.0.4855.1000 or later | Build 16.0.4390.1000 or later | Version 1602 (Build 6741.0000) or later | 1.22 or later | 15.20 or later| January 2017 | Version 1608 (Build 7601.6800) or later|
+| DialogApi 1.1  | Build 15.0.4855.1000 or later | Build 16.0.4390.1000 or later | Version 1602 (Build 6741.0000) or later | 1.22 or later | 15.20 or later | January 2017 | Version 1608 (Build 7601.6800) or later|
+| DialogApi 1.2  | N/A | N/A | See support<br>section below | 2.67 or later | 16.37 or later | June 2020 | N/A |
 
 >\* Users of the one-time purchase Office may not have accepted all patches and updates. If so, the DLL that Office uses to report its version in the UI may be greater than the versions listed here even if the updated DLLs needed to support DialogApi have not be installed on the user's computer. To ensure that the needed patch is installed, the user must go to the Office update list ([Office 2013 list](/officeupdates/msp-files-office-2013) or [Office 2016 list](/officeupdates/msp-files-office-2016)), search for **osfclient-x-none**, and install the listed patch.
 
+## Office on Windows (subscription) support
+
+The DialogApi 1.2 requirement set is supported in the Consumer Channel version 2005 (build, 12827.20268 or greater). For Office on Windows the feature is also supported in the Semi-Annual Channel and Monthly Enterprise Channel builds available June 9th, 2020 or later. The minimum supported builds for each channel are as follows:  
+
+|Channel | Version | Build|
+|:-----|:-----|:-----|
+|Current Channel | 2005 or greater | 12827.20160 or greater|
+|Monthly Enterprise Channel | 2004 or greater | 12730.20430 or greater|
+|Monthly Enterprise Channel | 2003 | 12624.20708 or greater|
+|Semi-Annual Enterprise Channel | 2002 or greater | 12527.20720 or greater|
```

Comment:
Just curious as to why the build number varies per channel. Isn't a feature usually tied to a specific build number and when the build is released based on the channel?

Reply:
Yes. It's nutz. Welcome to my world! Both 2-way dialog (and enable/disable) decided to GA without first completing the update of something called "source of truth". A side effect of that decision is that the feature is turned-on in different builds depending on channel. This table will become unnecessary in a few months.

Reply:
That reassuring. If we can eventually remove this table, can we make sure there's a ADO item tracking that? Maybe set the iteration path to whenever the final semi-annual channel update will go out?

Reply:
I'll do that.